tests/test_rand.c

```diff
@@ -20,6 +20,8 @@
 /* include rand.c to access private structures and functions */
 #include "rand.c"
```

Comment:
Please `#undef USE_GETRANDOM` before including `rand.c`. It is more important to verify DRBG implementation. Most checks (including CI) are performed on Linux systems, so we can miss broken patches. If you want to check `getrandom()` variant as well, create different test which will be stub if USE_GETRANDOM undefined.

Reply:
added a test with `getrandom()` disabled